apps/autonolas-registry/common-util/functions/requests.ts

```diff
@@ -0,0 +1,80 @@
+import { Address, TransactionReceipt } from 'viem';
+
+import { notifyError, sendTransaction } from '@autonolas/frontend-library';
+
+import { getEstimatedGasLimit } from 'libs/util-functions/src';
+
+import { SUPPORTED_CHAINS } from 'common-util/Login';
+
+import { DEFAULT_SERVICE_CREATION_ETH_TOKEN_ZEROS } from '../../util/constants';
+import { RPC_URLS, getDispenserContract, getServiceOwnerMultisigContract } from '../Contracts';
+import { checkIfGnosisSafe, getEthersProvider } from './index';
+
+const FALLBACK_HANDLER_STORAGE_SLOT =
+  '0x6c9a6c4a39284e37ed1cf53d337577d14212a4870fb976a4366c693b939918d5';
+
+/**
+ * function to check the owner address can mint.
+ * BE code: https://github.com/valory-xyz/autonolas-registries/pull/54#discussion_r1031510182
+ * @returns {Promise<boolean>} true if the owner address can mint
+ */
+export const checkIfERC721Receive = async (account: Address, ownerAddress: Address) => {
+  const provider = getEthersProvider();
+  const isSafe = await checkIfGnosisSafe(account, provider);
+
+  if (isSafe) {
+    try {
+      const contract = getServiceOwnerMultisigContract(account);
+      const threshold = await contract.methods.getThreshold().call();
+      const owners = await contract.methods.getOwners().call();
+
+      if (Number(threshold) > 0 && owners.length > 0) {
+        // TODO: check and fix error: Property 'getStorageAt' does not exist on type 'JsonRpcProvider | FallbackProvider'.
```

Comment:
pls take a look at this, ts says that there's `getStorageAt` on provider, but didn't have much time to investigate if it's safe to switch to `provider.getStorage` or the impact of this. Maybe we should plan the fix 🤔